src/components/FloatingLogo.tsx

```diff
@@ -21,15 +21,16 @@ import styled from 'styled-components'
 import { ReactComponent as AlephiumLogoSVG } from '../images/alephium_logo_monochrome.svg'
 import { deviceBreakPoints } from '../style/globalStyles'
 
-const FloatingLogo = styled(AlephiumLogoSVG)`
+const FloatingLogo = styled(AlephiumLogoSVG)<{ position?: 'top' | 'bottom' }>`
   position: absolute;
-  top: 50px;
   left: var(--spacing-5);
-  width: 60px;
-  height: 60px;
+  width: 35px;
```

Comment:
Despite the change in width, the actual size of the logo didn't change. This simply fixes alignment issues.